LibGAP.jl/test/basics.jl

```diff
@@ -0,0 +1,7 @@
+@testset "basics" begin
+    @test GAP.CSTR_STRING(GAP.GAPFuncs.String(GAP.GAPFuncs.PROD(2^59,2^59))) == "332306998946228968225951765070086144"
+
+    l = GAP.to_gap([1,2,3])
+
+    @test l[1] == 1
```

Comment:
You could also add a length test (we could even add a generic `length` method for MPtr